<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/icon/phonebook.svg</normaloff>:/icons/icon/phonebook.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color: &quot;transparent&quot;;

}
QToolButton
{
        background-color: transparent;
        border:2px solid transparent;
        border-radius: 5px;
}

QToolButton::hover
{
background-color:transparent;
border:1px solid rgb(204, 0, 0);
border-radius: 5px;
}
QToolButton::pressed
{
background-color: rgb(243, 246, 97);
border:1px solid rgb(164, 0, 0);
border-radius: 5px;
}

QHeaderView::section { background-color:rgb(252, 233, 79); }

QTableView {
        alternate-background-color:rgb(252, 175, 62);
        background-color:transparent;
        gridline-color: rgb(0, 0, 0);
        font-size: 14pt;
        font-family :&quot;Times New Roman&quot;;
		
}

QTableView:item:selected
{
        font-size: 16pt;
        color:rgb(204, 0, 0);
}

QTableView:item:hover
{
        font-size: 14pt;
 		color:rgb(204, 0, 0);}

QTableView:item
{
        color: rgb(0, 0, 0);
}

QTableView QTableCornerButton::section {
        background-color:rgb(252, 233, 79);
}

QTableView QScrollBar::handle:horizontal {
        background-color:rgb(252, 233, 79); 
}
QTableView QScrollBar::handle:vertical {
        background-color:rgb(252, 233, 79); 
}

QMessageBox
{
	background-color: rgb(255, 255, 255);
    font-family :&quot;Times New Roman&quot;;
     font-size: 14pt;


}
QMessageBox QPushButton{
	background-color: rgb(255, 255, 255);
    font-family :&quot;Times New Roman&quot;;
    font-size: 14pt;
}

QPushButton
{
   border: 0.5px solid rgb(0, 0, 0);
   border-top-right-radius: 15px;
   border-bottom-right-radius: 15px;
   border-top-left-radius: 0px;
   border-bottom-left-radius: 0px; 
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="wgt_base" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#wgt_base
{
border-image: url(:/images/images/background.svg) 0 0 0 0 stretch stretch;
}


</string>
      </property>
      <widget class="QPushButton" name="pb_add">
       <property name="geometry">
        <rect>
         <x>473</x>
         <y>110</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(138, 226, 52);
</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_edit">
       <property name="geometry">
        <rect>
         <x>473</x>
         <y>160</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(252, 233, 79);</string>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_remove">
       <property name="geometry">
        <rect>
         <x>473</x>
         <y>210</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 41, 41);</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_save">
       <property name="geometry">
        <rect>
         <x>473</x>
         <y>260</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(115, 238, 183);</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_load">
       <property name="geometry">
        <rect>
         <x>473</x>
         <y>310</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(173, 127, 168);</string>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_exp_csv">
       <property name="geometry">
        <rect>
         <x>473</x>
         <y>360</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 50, 131);</string>
       </property>
       <property name="text">
        <string>Export CSV</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tb_close">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>50</y>
         <width>26</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/icons/icon/close.svg</normaloff>:/icons/icon/close.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="tb_minimize">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>50</y>
         <width>26</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/icons/icon/minimize.svg</normaloff>:/icons/icon/minimize.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>341</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>30</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">        font-family :&quot;Times New Roman&quot;;
</string>
       </property>
       <property name="text">
        <string>Phone Book</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>361</width>
         <height>471</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>27</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>32</number>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
